presto-spi/src/test/java/io/prestosql/spi/block/TestLongArrayBlockEncoding.java

```diff
@@ -14,29 +14,24 @@
 package io.prestosql.spi.block;
 
 import io.prestosql.spi.type.Type;
-import org.testng.annotations.Test;
+
+import java.util.Random;
 
 import static io.prestosql.spi.type.BigintType.BIGINT;
 
 public class TestLongArrayBlockEncoding
         extends BaseBlockEncodingTest<Long>
 {
-    @Test
-    public void testRoundTripNoNull()
-    {
-        roundTrip(1L, 2L, 3L, 4L);
-    }
-
-    @Test
-    public void testRoundTripWithNull()
+    @Override
+    protected void write(BlockBuilder blockBuilder, Long value)
     {
-        roundTrip(null, null, 1L, null, 3L, null, null);
```

Comment:
this was specifically testing `null` at beginning and end as it requires special treatment in optimized block encoding

Reply:
I'll add a test case for that then.